presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -187,6 +187,9 @@ public Integer call()
             catch (TimeoutExceededException ignored) {
                 log.error("Test execution exceeded timeout of %s", timeout);
             }
+            catch (IllegalArgumentException e) {
+                log.error("Invalid timeout %s for test execution", timeout);
```

Comment:
getting IAE here, can you be sure it's about timeout being incorrect?
better add an explicit check above
```
long timeoutMillis = timeout.toMillis();
if (timeoutMillis == 0) {
    log.eror("Invalid timeout or timeout exhausted");
    return ExitCode.SOFTWARE;
}

....
with(Timeout.of(java.time.Duration.ofMillis(timeoutMillis))
```